client/src/components/Backend/screens/UserStructure/UserStructure.component.tsx

```diff
@@ -56,7 +56,7 @@ export const UserStructureComponent = (props: Props) => {
       );
     }
     window.scrollTo(0, 0);
-  }, [dispatch]);
+  }, [dispatch, userStructure]);
```

Comment:
Like for the userContributions, this is causing an infinite loop. 
You can see it by going to the Structure tab of the backoffice (you need to be a user with a structure for this)

Reply:
Again I can't reproduce this locally.